setup.py

```diff
@@ -72,9 +72,9 @@ def run_tests(self):
         'Flask',
         'Flask-RESTful>=0.2.12',
         'blinker',
-        'redis',
         'six',
         'pillow',
+        'ordereddict',
```

Comment:
if it's only for Python 2.6 then please do something similar to https://github.com/inveniosoftware/kwalitee/blob/master/setup.py#L33-L52. Thanks